tests/drivers/build_all/sensors_i_z.conf

```diff
@@ -12,6 +12,8 @@ CONFIG_LIS3DH=y
 CONFIG_LIS3MDL=y
 CONFIG_LPS22HB=y
 CONFIG_LPS25HB=y
+CONFIG_LSM303DLHC_ACCEL=y
+CONFIG_LSM303DLHC_MAGN=y
```

Comment:
Move the `tests/drivers/build_all/dts.fixup` changes into this commit

Reply:
Done